server/routes/discounts.router.js

```diff
@@ -139,6 +139,28 @@ router.put("/", rejectUnauthenticated, rejectUnauthorizedUser, (req, res) => {
     });
 }); // End edit discount PUT
 
+// This PUT will toggle the is_shown for a discount by id
+router.put("/active", rejectUnauthenticated, rejectUnauthorizedUser, (req, res) => {
```

Comment:
I believe this should have rejectNonAdmin, not rejectUnauthorized.

Reply:
Changed